packages/ozone/src/db/migrations/20240208T213404429Z-add-lang-column-to-moderation-subject.ts

```diff
@@ -0,0 +1,15 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_subject_status')
+    .addColumn('langs', 'jsonb')
```

Comment:
would the array datatype be easier here? I don't have a strong opinion

Reply:
hmm... do you have anything written down about the flag approach? just trying to imagine the design for this. do we just have a `flags` array column which stores arbitrary string flags? or do we need a more elaborate, many to many relationship set up to normalize the flags?
If we go with the former, it's straight forward of course.

Reply:
We already use `jsonb` for structured arrays in a couple places (i.e. blob cids), I'd be inclined to stick with that just by convention.